cds_ils/importer/documents/importer.py

```diff
@@ -257,6 +257,29 @@ def _validate_match_identifiers(self, existing_document):
         # not valid - will create a new document
         return False
 
+    def _validate_volumes(self, existing_document):
+        """Validate serial volumes match."""
+        import_doc_serial = self.json_data.get("_serial", None)
+        existing_doc_serials = existing_document.relations.get('serial', [])
+        if import_doc_serial and existing_doc_serials:
+            import_volume = import_doc_serial.get("volume")
+            import_serial_title = import_doc_serial["title"].lower()
+
+            same_serial, same_volume, both_have_volumes = False, False, False
+
+            for serial in existing_doc_serials:
+                existing_volume = serial.get["volume"]
+                existing_title = serial["record_metadata"]["title"].lower()
+                if existing_title == import_serial_title:
+                    same_serial = True
+                    both_have_volumes = all([import_volume, existing_volume])
```

Comment:
Maybe the usage of `all` + `list` is a bit too much here?
```suggestion
                    both_have_volumes = import_volume and existing_volume
```
If not defined, they should be `None` I think.